.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -19,7 +19,7 @@ A clear and concise description of what the bug is.
 4. See error
 
 ### Environment
-* Bugsnag version:
+* Bugsnag Go Negroni version:
```

Comment:
```suggestion
* Bugsnag Go version:
* Bugsnag Go Negroni version:
```

Ah, we probably want both here since they are now separate (and versioned separately)